sdcm/sct_config.py

```diff
@@ -1087,12 +1087,12 @@ def __init__(self):
         files = anyconfig.load(list(config_files))
         anyconfig.merge(self, files)
 
-        regions_data = self.get('regions_data', {})
+        regions_data = self.get('regions_data') or {}
         if regions_data:
             del self['regions_data']
 
         # 2.2) load the region data
-        region_names = self.get('region_name', '').split()
+        region_names = self.get('region_name') or ''.split()
```

Comment:
need to run .split on the results of `self.get('region_name')`

Reply:
fixed